package.json

```diff
@@ -77,9 +77,9 @@
     },
     "lint-staged": {
         "*.{js,mjs,ts}": "eslint --cache",
-        "*.{css,js,json,md,mjs,ts,yaml,yml}": "prettier --write",
-        "{packages/**/package.json,scripts/tasks/check-and-rewrite-package-json.js}": "node ./scripts/tasks/check-and-rewrite-package-json.js",
-        "{LICENSE-CORE.md,**/LICENSE.md,yarn.lock,scripts/tasks/generate-license-files.js,scripts/shared/bundled-dependencies.js}": "node ./scripts/tasks/generate-license-files.js",
+        "*.{css,js,json,md,mjs,ts,yaml,yml}": "prettier --check",
+        "{packages/**/package.json,scripts/tasks/check-and-rewrite-package-json.js}": "node ./scripts/tasks/check-and-rewrite-package-json.js --test",
+        "{LICENSE-CORE.md,**/LICENSE.md,yarn.lock,scripts/tasks/generate-license-files.js,scripts/shared/bundled-dependencies.js}": "node ./scripts/tasks/generate-license-files.js --test",
```

Comment:
This changes `git commit` to only validate files pre-commit, rather than edit them. The auto-changing annoyed me when I tried to test this change!